python/cudf/bindings/cudf_cpp.pyx

```diff
@@ -72,16 +72,20 @@ cdef gdf_dtype get_dtype(dtype):
 
 
 cdef get_scalar_value(gdf_scalar scalar):
+    """
+    Returns typed value from a gdf_scalar
+    0-dim array is retuned if dtype is date32/64, timestamp
+    """
     return {
         GDF_FLOAT64: scalar.data.fp64,
         GDF_FLOAT32: scalar.data.fp32,
         GDF_INT64:   scalar.data.si64,
         GDF_INT32:   scalar.data.si32,
         GDF_INT16:   scalar.data.si16,
         GDF_INT8:    scalar.data.si08,
-        GDF_DATE32:  scalar.data.dt32,
-        GDF_DATE64:  scalar.data.dt64,
-        GDF_TIMESTAMP: scalar.data.tmst,
+        GDF_DATE32:  np.array(scalar.data.dt32).astype('datetime64[D]'),
+        GDF_DATE64:  np.array(scalar.data.dt64).astype('datetime64[ms]'),
+        GDF_TIMESTAMP: np.array(scalar.data.tmst).astype('datetime64[ns]'),
```

Comment:
Is `GDF_TIMESTAMP` always nanosecond resolution in libcudf? My understanding is that the resolution can be arbitrary and is an attribute of the `gdf_column` object.

Reply:
Yeah, the resolution is technically a separate attribute of the `gdf_column`. Nothing in `libcudf` actually uses or respects that resolution right now :)

Reply:
Thanks @jrhemstad, will just merge this for now and can be resolved when we actually begin using `GDF_TIMESTAMP`